### PR TITLE
Adapt methods to add score times the level and to increase level when…

### PR DESCRIPTION
**Pull request by wroel**
With this implementation the score needed to reach new level is multiplated with the level as well as the score awarded for a connected line.

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/semgr8/model/Game.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ public void updateScore() {
      * @param s the integer score to be added.
      */
     public void addScore(int s) {
-        score.addScore(s);
+        score.addScore(s * level.getLevel());
```
Comment (ghyl) [inline src/main/java/nl/tudelft/ti2206/semgr8/model/Game.java]: Multiplying the added score with the level seems like way too much of a bonus to me. You could better do something like: `s + Math.ceil(s * level.getLevel() * 0.1)` or something. Didn't play the game like this though.

**Comment by ghyl** [comment]
can merge when comment is processed

**Comment by wroel** [comment]
The score added per level is still in proportion with the score needed to reach a new level in comparison with before this implementation. So only higher scores are achieved, the gameplay isn't changed.

**Comment by ghyl** [comment]
Ok, then I'll merge it!

**Pull request merged**